packages/cosmos_ui_components/lib/cosmos_theme.dart

```diff
@@ -161,6 +162,7 @@ class CosmosColorsData extends Equatable {
   final Color chipBackground;
   final Color positiveText;
   final Color link;
+  final Color clearButtonBg;
```

Comment:
this color name is again too specific, we won't be able to include all the colors inside the designs into theme and should instead "generalize" the pallete as much as possible, also I can't see this color used anywhere

Reply:
Removed.